spec/version_parser_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rspec'
+require 'version_parser'
```

Comment:
require spec_helper instead of the above lines, include version_parser in the spec helper.  (the spec helper is in master and should be merged into this branch)